app/views/homepage/_moves.html.slim

```diff
@@ -18,7 +18,7 @@
           span.strong-text
             | #{move.detainee.surname}
           span
-            |  #{move.detainee.forenames}
+            | #{move.detainee.forenames}
```

Comment:
In order to preserve the white space here, we can use slim:
https://github.com/slim-template/slim#verbatim-text-with-trailing-white-space-

```
span.strong-text
  ' #{move.detainee.surname}
span
  | #{move.detainee.surname}
```